docs/resources/fleetplan.md

```diff
@@ -323,4 +327,17 @@ Optional:
 - `commands` (List of String) Specifies the commands to execute within the container.
 - `env` (Map of String) Pass variables to the post-hook script that will be available at runtime.
   - Example: `"key1:value1"`
+- `cpu_limit_milli` (string) CPU Limit in millicores
+  - Example: "1000"
+- `memory_limit_mb` (string) Maximum MB to be used by the container
+  - Example: "100"
+- `working_dir_path` (string) Path where the process should run
+  - "/opt/"
```

Comment:
Change it to Example: "/var/"

Don't we need to add same config for prehook as well?